packages/@webex/plugin-meetings/test/unit/spec/roap/request.ts

```diff
@@ -1,3 +1,4 @@
+import 'jsdom-global/register';
```

Comment:
Is this change required as part of this PR?

Reply:
Yes, this was required as we were getting `window is not defined` error. Discussed with Anna as well, this is the fix.